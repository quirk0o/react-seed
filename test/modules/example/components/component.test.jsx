import React from 'react'
import {shallow} from 'enzyme'

import Component from 'modules/example/components/component'

describe('<Component />', function () {
  def('test', () => 'value')

  subject(function () {
    return shallow(<Component test={this.test} />)
  })

  it('renders', function () {
    expect(this.subject).toMatchSnapshot()
  })
})