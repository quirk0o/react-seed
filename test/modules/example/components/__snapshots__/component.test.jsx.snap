// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Component /> renders 1`] = `
<div>
  Component
</div>
`;
